emloop/tests/hooks/classification_metrics_test.py

```diff
@@ -30,6 +34,7 @@ def test_computing_metrics():
     epoch_data = get_epoch_data()
 
     hook.after_batch(stream_name='train', batch_data=epoch_data['train'])
+    hook.after_batch(stream_name='test', batch_data=epoch_data['test'])
```

Comment:
This isn't what I meant. We need to simulate multiple batches within a single stream.

We need to know that if we pass two batches `{gt: 0,1,0 pred: 0,0,1}` and `{gt: 1,1,0 pred: 1,0,1}`, the metrics would be computed from `{gt: 0,1,0,1,1,0 pred: 0,0,1,1,0,1}`.

Anyway, this line is good since we test that the stream data are not mixed with each other.